pkg/runtime/cli.go

```diff
@@ -155,7 +155,7 @@ func FromFlags() (*DaprRuntime, error) {
 			}
 			defer conn.Close()
 			namespace = os.Getenv("NAMESPACE")
-			globalConfig, configErr = global_config.LoadKubernetesConfiguration(*config, namespace, client)
+			globalConfig, _, configErr = global_config.LoadKubernetesConfiguration(*config, namespace, client)
```

Comment:
Why do we need to return original payload ?

Reply:
Dashboard displays it. Otherwise, I cannot reuse this method.

Reply:
Let me fix this, it needs to be in the standalone method, not this one.

Reply:
See new version now, please.